flux/src/flagger/canarydetails.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { K8s } from '@kinvolk/headlamp-plugin/lib';
+import { useParams } from 'react-router-dom';
+import { ConditionsTable, Link, MainInfoSection, NameValueTable, SectionBox } from '@kinvolk/headlamp-plugin/lib/components/common';
+import CanaryStatus from './canarystatus';
+import { ObjectEvents } from '../helpers';
+import Event, { KubeEvent } from '@kinvolk/headlamp-plugin/lib/K8s/event';
+import { SuspendAction, ResumeAction, RollbackCanary } from '../actions';
+import FlaggerAvailabilityCheck from './availabilitycheck';
+
+export default function CanaryDetails() {
+    const [canary] = K8s.ResourceClasses.CustomResourceDefinition.useGet('canaries.flagger.app');
+  
+    return (
+        <FlaggerAvailabilityCheck>
+         { canary && <CanaryDetailsRenderer resource={canary}/> }
```

Comment:
No loader in this case?

Reply:
In this case the FlaggerAvailabilityCheck takes care of it

Reply:
updated